docs/src/assertions.md

```diff
@@ -8,12 +8,14 @@ DocTestSetup = AbstractAlgebra.doctestsetup()
 We describe here various macros provided by AbstractAlgebra.
 
 ## Verbosity macros
-There is a list of symbols called *verbosity scopes* which represent keywords used to
-trigger some particular macros within the codes. Each of these verbosity scopes is 
-associated with a *verbosity level*, being set to $0$ by default. A verbosity macro
-is joined to a verbosity scope `S` and a value `k` (set to $1$ by default) such that,
+There is a (global) list of symbols called *verbosity scopes* which represent keywords used to
+trigger some verbosity macros within the code. Each of these verbosity scopes has its own
+integer *verbosity level*, which is set to $0$ by default. A verbosity macro
+must specify the verbosity scope `S` and optionally the trigger level `k` (defaulting to $1$) such that,
```

Comment:
Not quite sure I understand this change. What does it mean that a macro "must specify" something? Is this talking about implementing or invoking a macro? Now it sounds like implementing a macro.

Reply:
I think that makes it clear, thanks.